templates/conference/header/_homepage_jumbotron.html

```diff
@@ -2,14 +2,14 @@
 
 <div class="jumbotron">
     <div class="container text-center">
-        <img class="img-fluid" src="{% static 'img/ep2020-online-logo.png' %}"
-             alt="EuroPython 2020; July 23-26th 2020; Online"
-             title="EuroPython 2020; July 23-26th 2020; Online">
+        <img class="img-fluid" src="{% static 'img/ep2021-online-logo.png' %}"
+             alt="EuroPython 2021; July 26 - Aug. 1, 2021; Online"
```

Comment:
Same as above for all mentions of the date range.

Reply:
Fixed